doc/Requirements Specification/Parts/Discussion.lyx

```diff
@@ -263,6 +263,20 @@ gls{PCM}
 \end_layout
 
 \end_deeper
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/C40/"
+
+\end_inset
+
+ In the use of JaMopp the source code's Java version likely is restricted
+ by JaMopp.
+ The question is, if it is useful to further projects to get independent
```

Comment:
I don't think we can mention something with "The question is..." in the SRS.
We have to specify our software, not the problems with our software!!!